pallets/committee/src/lib.rs

```diff
@@ -328,7 +331,10 @@ decl_module! {
         ///
         /// # Errors
         /// * `NotAMember`, If the new coordinator `id` is not part of the committee.
-        #[weight = <T as Trait<I>>::WeightInfo::set_release_coordinator()]
+        #[weight = (
+            <T as Trait<I>>::WeightInfo::set_release_coordinator(),
+            DispatchClass::Operational
```

Comment:
The thing is that `CommitteeOrigin` is defined outside this pallet, so I would not want to make that assumption here.
I've explicitly marked it as `Operational` based on the purpose of that extrinsic, instead of any external info.
For instance, if we update or create a new config (like dev, testnet,...) where `CommitteOrigin == EnsureRoot`, you could call this function directly, and no changes will be done inside `committee` pallet.
So in our current configuration, that change is not mandatory, but I think it improves the readability.